tests/pytests/test_hybrid_groupby.py

```diff
@@ -0,0 +1,144 @@
+from RLTest import Env
+from includes import *
+from common import *
+import numpy as np
+from collections import Counter
+
+"""
+FT.HYBRID GROUPBY FLOW TESTS:
+===============================
+
+VECTOR SPACE LAYOUT:
+===================
+Query vector at origin [0, 0]
+
+doc:1, doc:2 and doc:3 are at distance 1 from the query vector [0,0]
+doc:4, doc:5 and doc:6 are at distance 2 from the query vector [0,0]
+doc:7, doc:8 and doc:9 are at distance 3 from the query vector [0,0]
+
+This allows controlling result set sizes with RADIUS:
```

Comment:
Since it's not documented now, can we add a note that we take L2distance^2 and not L2distance?